src/main/java/org/folio/oaipmh/Constants.java

```diff
@@ -2,6 +2,10 @@
 
 public final class Constants {
 
+  private Constants() {
+    throw new IllegalStateException("This class holds constants only");
```

Comment:
I never thought to do this, always just used an empty constructor, but I like it.